src/components/molecules/message/message.html

```diff
@@ -0,0 +1,4 @@
+<div class="message">
+  <h1 style="margin: 0" class="title">タイトルテキスト</h1>
+  <p style="margin: 0" class="explain">説明テキスト</p>
```

Comment:
```suggestion
  <h2 class="title">タイトルテキスト</h1>
  <p class="explain">説明テキスト</p>
```
tomuさんのご指摘の通り、h1タグはページ全体で一度しか使ってはいけないことになっているので、今回のように使い回す前提のコンポーネントには不向きです。
参考：https://developer.mozilla.org/ja/docs/Web/HTML/Element/Heading_Elements

また、styleタグは使用せず、CSSで見た目を調節するようにしましょう。